Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -46,16 +46,28 @@ import BraintreeCore
     /// Used for sending the type of flow, universal vs deeplink to FPTI
     private var linkType: LinkType?
 
+    private var universalLink: URL?
+
     // MARK: - Initializer
 
-    /// Creates an Apple Pay client
+    /// Creates an Venmo client
```

Comment:
small nit (take it or leave it): 
```suggestion
    /// Creates a Venmo client
```